pkgs/by-name/li/lifelines/package.nix

```
@@ -13,25 +12,15 @@
 
 stdenv.mkDerivation {
   pname = "lifelines";
-  version = "unstable-2019-05-07";
+  version = "unstable-2025-01-05";
```

```suggestion
  version = "0-unstable-2025-01-05";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

the commit also needs to be renamed